server/controllers/medical/snis.js

```diff
@@ -290,11 +290,24 @@ function getReport (req, res) {
 	}
 }
 
+function health_zones (req, res, next) {
```

Comment:
Please use camelCase instead of snake_case for JavaScript variables, functions, and objects.